File: JavaScript/8-kyu/7. Return Negative/index.test.js
Test.assertEquals(makeNegative(42), -42);
Test.assertEquals(makeNegative(-9), -9);
Test.assertEquals(makeNegative(0), 0);
Test.assertEquals(makeNegative(1), -1);
Test.assertEquals(makeNegative(-1), -1);

Test.assertEquals(makeNegative(0.12), -0.12);
Test.assertEquals(makeNegative(0.00001), -0.00001);
Test.assertEquals(makeNegative(-0.00001), -0.00001);

var rnd = Math.floor(Math.random() * 101);
Test.assertEquals(makeNegative(rnd), -rnd);

rnd = -Math.floor(Math.random() * 101);
Test.assertEquals(makeNegative(rnd), rnd);

File: Python/8-kyu/7. Return Negative/index.test.py
Test.assert_equals(make_negative(42),-42)
Test.assert_equals(make_negative(-9),-9)
Test.assert_equals(make_negative(0),0)
Test.assert_equals(make_negative(1),-1)
Test.assert_equals(make_negative(-1),-1)

from random import randint as rnd

for _ in range(10):
    number = rnd(1, 1000)
    Test.assert_equals(make_negative(number),-abs(number))
    number = rnd(-1000,0)
    Test.assert_equals(make_negative(number),-abs(number))